src/runtime_src/core/include/xclbin.h

```diff
@@ -341,6 +353,21 @@ extern "C" {
                                  //         m_int_enable   : Bit  - 0x0000_0001;
                                  //         m_interrupt_id : Bits - 0x0000_00FE;
                                  //         m_ip_control   : Bits = 0x0000_FF00;
+                                 // proeprties is also used for ps kernel (i.e. -add-pskernel)
+       
+            // PS Kernels
+            // m_type: IP_PS_KERNEL
+            //         m_subtype: 0 - PS, 1 - DPU
```

Comment:
The m_type and m_functional comments one line 360 & 361 are not needed.

Reply:
comments removed